### ISSUE TITLE
Carthage integration

### ISSUE THREAD

**Issue by MetalheadSanya** [rambler-ios/ViperMcFlurry#27]
Version 1.5.1 does not build with latest Carthage.

```
CompileC ~/Library/Developer/Xcode/DerivedData/ViperMcFlurry-fixoxtjhfnbvqbcmndjrdxijtiic/Build/Intermediates/ViperMcFlurry.build/Release-iphoneos/ViperMcFlurry.build/Objects-normal/arm64/RamblerViperModuleFactory.o /Users/alexanderzalutskiy/Work/Gazelkin/Carthage/Checkouts/ViperMcFlurry/Code/IntermoduleDataTransfer/RamblerViperModuleFactory.m normal arm64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
CompileC ~/Library/Developer/Xcode/DerivedData/ViperMcFlurry-fixoxtjhfnbvqbcmndjrdxijtiic/Build/Intermediates/ViperMcFlurry.build/Release-iphoneos/ViperMcFlurry.build/Objects-normal/armv7/RamblerViperOpenModulePromise.o /Users/alexanderzalutskiy/Work/Gazelkin/Carthage/Checkouts/ViperMcFlurry/Code/IntermoduleDataTransfer/RamblerViperOpenModulePromise.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
CompileC ~/Library/Developer/Xcode/DerivedData/ViperMcFlurry-fixoxtjhfnbvqbcmndjrdxijtiic/Build/Intermediates/ViperMcFlurry.build/Release-iphoneos/ViperMcFlurry.build/Objects-normal/armv7/RamblerViperModuleFactory.o /Users/alexanderzalutskiy/Work/Gazelkin/Carthage/Checkouts/ViperMcFlurry/Code/IntermoduleDataTransfer/RamblerViperModuleFactory.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
CompileC ~/Library/Developer/Xcode/DerivedData/ViperMcFlurry-fixoxtjhfnbvqbcmndjrdxijtiic/Build/Intermediates/ViperMcFlurry.build/Release-iphoneos/ViperMcFlurry.build/Objects-normal/arm64/RamblerViperOpenModulePromise.o /Users/alexanderzalutskiy/Work/Gazelkin/Carthage/Checkouts/ViperMcFlurry/Code/IntermoduleDataTransfer/RamblerViperOpenModulePromise.m normal arm64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
CompileC ~/Library/Developer/Xcode/DerivedData/ViperMcFlurry-fixoxtjhfnbvqbcmndjrdxijtiic/Build/Intermediates/ViperMcFlurry.build/Release-iphoneos/ViperMcFlurry.build/Objects-normal/arm64/UIViewController+RamblerViperModuleTransitionHandlerProtocol.o /Users/alexanderzalutskiy/Work/Gazelkin/Carthage/Checkouts/ViperMcFlurry/Code/IntermoduleDataTransfer/UIViewController+RamblerViperModuleTransitionHandlerProtocol.m normal arm64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
CompileC ~/Library/Developer/Xcode/DerivedData/ViperMcFlurry-fixoxtjhfnbvqbcmndjrdxijtiic/Build/Intermediates/ViperMcFlurry.build/Release-iphoneos/ViperMcFlurry.build/Objects-normal/armv7/UIViewController+RamblerViperModuleTransitionHandlerProtocol.o /Users/alexanderzalutskiy/Work/Gazelkin/Carthage/Checkouts/ViperMcFlurry/Code/IntermoduleDataTransfer/UIViewController+RamblerViperModuleTransitionHandlerProtocol.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
```

**Comment by MetalheadSanya** [comment]
merge requires with fix problem #28

**Comment by etolstoy** [comment]
Version 1.5.2 is released.
